packages/lesmis/src/views/MapView.js

```diff
@@ -71,7 +71,7 @@ const getDefaultTileSet = () => {
 
 export const MapView = () => {
   const { entityCode } = useUrlParams();
-  const [selectedYear, setSelectedYear] = useState(DEFAULT_DATA_YEAR);
+  const [selectedYear, setSelectedYear] = useUrlSearchParam('year', DEFAULT_DATA_YEAR);
```

Comment:
@reviewer I would be interested to get your input on the name `year` for this param. I thought about something more descriptive such as `dashboardYear` but the year is used for the map and the dashboard so I left it as year.

Reply:
It fully depends on whether the "global year selector" is a shared state across dashboard and map tabs. Personally I would have expected that they were independently controlled, given that they're in quite different places rather than in some common place that doesn't change between the two. For that reason, I would go for `dashboardYear` + `mapYear`. If the state should be shared between dashboard and map overlay, then `year` is the right key.

I'll ask in the channel

Reply:
https://beyondessential.slack.com/archives/G01NEKFK76C/p1621813723006100?thread_ts=1621566541.000300&cid=G01NEKFK76C

Go for `year` :-)